src/services/reading-challenges.service.ts

```diff
@@ -22,6 +22,15 @@ export const getReadingChallengeById = async (id: number) => {
   return readingChallenge;
 };
 
```

Comment:
@MOHAMEDTAREEK Regarding `getReadingChallengeById`, the `id` parameter is not optional _aka_ doesn't have a `?` at the end of its name, so there's no need to check if it doesn't exist (lines 16 to 18).